### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by s-a-y** [BitGo/BitGoJS#610]
bitgo: v9.0.1
node 12.13.1 and 10.16
typescript: 3.6.3

```
node_modules/bitgo/dist/types/src/v2/coins/algo.d.ts:7:20 - error TS2440: Import declaration conflicts with local declaration of 'TransactionExplanation'.

7 import { BaseCoin, TransactionExplanation, KeyPair, ParseTransactionOptions, ParsedTransaction, VerifyAddressOptions, VerifyTransactionOptions, SignedTransaction } from '../baseCoin';
                     ~~~~~~~~~~~~~~~~~~~~~~

Found 1 error(s).
```

replacing `TransactionExplanation` by `TransactionExplanation as TransactionExplanationBase` and then extending TransactionExplanationBase helps to overcome the problem

**Comment by mathieu-lgo** [comment]
My app is not building, because of this error please fix it quickly.

**Comment by s-a-y** [comment]
> My app is not building, because of this error please fix it quickly.

I'm using a workaround, modified the file to fix the error, saved it into separate folder and copying it over on postinstall

**Comment by mathieu-lgo** [comment]
Thank you @s-a-y for the workaround !

It's very sad that bitgo didn't fix this issue, it look like stupid mistake...

**Comment by tylerlevine** [comment]
Thanks for bumping this back up, we'll get this fixed in the next release.

Also, does it still error with `skipLibCheck: true` in your tsconfig.json?

**Comment by mathieu-lgo** [comment]
Happy to read that you will fix it !
Yes using `skipLibCheck: true` it works, this is useful for development but is not very safe tu compile an app with this option for production you have no idea if something else is not working 🙈

**Comment by s-a-y** [comment]
Fixed in #644
